save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/CreateOrganizationView.kt

```diff
@@ -61,42 +56,24 @@ external interface OrganizationSaveViewState : State {
 @JsExport
 @OptIn(ExperimentalJsExport::class)
 class CreateOrganizationView : AbstractView<Props, OrganizationSaveViewState>(true) {
-    private val fieldsMap: MutableMap<InputTypes, String> = mutableMapOf()
-
     init {
         state.isErrorWithOrganizationSave = false
         state.errorMessage = ""
-        state.isValidOrganizationName = true
-    }
-
-    private fun changeFields(fieldName: InputTypes, target: Event, isOrganization: Boolean = true) {
-        val tg = target.target as HTMLInputElement
-        if (isOrganization) fieldsMap[fieldName] = tg.value else fieldsMap[fieldName] = tg.value
+        state.organizationDto = OrganizationDto.empty
     }
 
     @Suppress("UnsafeCallOnNullableType", "TOO_LONG_FUNCTION", "MAGIC_NUMBER")
     private fun saveOrganization() {
-        if (!isValidInput()) {
-            return
-        }
-        val organizationName = fieldsMap[InputTypes.ORGANIZATION_NAME]!!.trim()
-        val dateCreated = LocalDateTime(1970, Month.JANUARY, 1, 0, 0, 1)
-
-        val newOrganizationRequest = Organization(organizationName, OrganizationStatus.CREATED, null, dateCreated, null)
-        val headers = Headers().also {
-            it.set("Accept", "application/json")
-            it.set("Content-Type", "application/json")
-        }
         scope.launch {
             val responseFromCreationOrganization =
                     post(
                         "$apiUrl/organization/save",
-                        headers,
-                        Json.encodeToString(newOrganizationRequest),
+                        jsonHeaders,
+                        Json.encodeToString(state.organizationDto),
                         loadingHandler = ::classLoadingHandler,
                     )
             if (responseFromCreationOrganization.ok) {
-                window.location.href = "${window.location.origin}#/${organizationName.replace(" ", "%20")}/"
+                window.location.href = "${window.location.origin}#/${state.organizationDto.name.replace(" ", "%20")}/"
```

Comment:
`String.replace(" ", "%20")` can be extracted to an Utils file

Reply:
As far as we forbade using whitespaces in names, we do not need it anymore. The only reason why it is still here is that we have in dev db examples with whitespaces. In next PR I will change `project-insert.xml` and `organization.csv` in order to get rid of those examples. In the same PR I will remove this replace